spk/fuse-exfat/src/installer.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# Package
+PACKAGE="exfat"
+DNAME="exfat"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put exfat & fusermount in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bin/mount.exfat /usr/local/bin/mount.exfat
+	ln -s ${INSTALL_DIR}/bin/fusermount /usr/local/bin/fusermount
```

Comment:
This caught my eye, and I'm not talking about the indenting issue :) The `sshfs` package creates the same symlink, but to a different location. That's no good, because it means unexpected results if both packages are installed/uninstalled. Does this symlink need to exist? Note that the symlinks are only a convenience option, not a necessity.